src/OrchardCore.Modules/OrchardCore.Media/Razor/OrchardRazorHelperExtensions.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using OrchardCore.DisplayManagement.Razor;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace OrchardCore.Media.Razor
```

Comment:
Use `OrchardCore.DisplayManagement.Razor` such that the extension appears without having to add an extra `@using`